### DISCUSSION TITLE
Question about the correct CSV format for dividend imports

### DISCUSSION THREAD

**Discussion by Krabtus** [ghostfolio/ghostfolio#3794] [Q&A] [answered]
Hi everyone,

I'm trying to import dividend payments using the following CSV format:

`Date,Code,DataSource,Currency,Price,Quantity,Action,Fee,Account`

Unfortunately, I'm having trouble because the system doesn't recognize the data correctly when I want to import dividends using these columns. Could someone explain to me exactly how the columns need to be structured so that the import works successfully?

Thank you in advance for your support!

**Comment by dtslvr** [comment, marked as answer]
You can find an example here: https://github.com/ghostfolio/ghostfolio/blob/main/test/import/ok.csv

**Comment by Krabtus** [comment]
Perfect, thanks for your tip. It works!
